src/safeposix/syscalls/fs_constants.rs

```diff
@@ -76,14 +76,14 @@ pub const F_GETLEASE: i32 = 1025;
 pub const F_NOTIFY: i32 = 1026;
 
 //File types for open/stat etc.
-pub const S_IFBLK: i32 = 0o60000;
-pub const S_IFCHR: i32 = 0o20000;
+pub const S_IFBLK: i32 = 0o60000; 
+pub const S_IFCHR: i32 = 0o20000; 
 pub const S_IFDIR: i32 = 0o40000;
-pub const S_IFIFO: i32 = 0o10000;
-pub const S_IFLNK: i32 = 0o120000;
-pub const S_IFREG: i32 = 0o100000;
-pub const S_IFSOCK: i32 = 0o140000;
-pub const S_FILETYPEFLAGS: i32 = 0o170000;
+pub const S_IFIFO: i32 = 0o10000;   
+pub const S_IFLNK: i32 = 0o120000;  
+pub const S_IFREG: i32 = 0o100000;  
+pub const S_IFSOCK: i32 = 0o140000;        
+pub const S_FILETYPEFLAGS: i32 = 0o170000; 
```

Comment:
You added a bunch of trailing spaces here, it's not a huge deal but if you can remove them in your commit that would be good

Reply:
On it!